config/enterprise.go

```diff
@@ -0,0 +1,104 @@
+package config
```

Comment:
Manually reviewed this file against https://github.com/mesosphere/dcos-signal-enterprise/blob/master/config/enterprise.go and it looks good.